### PR TITLE
Configure kuiper

### PR DESCRIPTION
**Pull request by mirceacaprioru**

### CODE REVIEW

File: stage2/01-sys-tweaks/01-run.sh [hunk 1]
```diff
@@ -17,6 +17,8 @@ install -m 644 files/x11vnc.service	"${ROOTFS_DIR}/lib/systemd/system/"
 install -d				"${ROOTFS_DIR}/home/${FIRST_USER_NAME}/.vnc"
 install -m 644 files/passwd		"${ROOTFS_DIR}/home/${FIRST_USER_NAME}/.vnc/"
 
+cp -f "${ROOTFS_DIR}/usr/share/doc/avahi-daemon/examples/ssh.service" "${ROOTFS_DIR}/etc/avahi/services/"
```
Comment (commodo) [inline stage2/01-sys-tweaks/01-run.sh:20]: what if you use the `install` command like above?
that also makes sure the proper chmod is used;

File: stage6/05-adi-release-notes/00-run.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -e
+
+install -m 755 files/os-release	"${ROOTFS_DIR}/usr/lib/"
+
+
```
Comment (commodo) [inline stage6/05-adi-release-notes/00-run.sh:5]: extra lines?

File: stage6/files/os-release [hunk 3]
```diff
@@ -0,0 +1,10 @@
+PRETTY_NAME="Kuiper GNU/Linux 10 (buster)"
+NAME="Kuiper GNU/Linux"
+VERSION_ID="10"
+VERSION="10 (buster)"
+VERSION_CODENAME=buster
+ID=raspbian
+ID_LIKE=debian
+HOME_URL="https://github.com/analogdevicesinc/adi-kuiper-gen"
+SUPPORT_URL="https://ez.analog.com/linux-software-drivers/"
+
```
Comment (commodo) [inline stage6/files/os-release:10]: empty line?

**Comment by commodo** [comment]
rest looks good

File: stage2/01-sys-tweaks/01-run.sh (hunk 1 above)
Comment (mirceacaprioru) [reply stage2/01-sys-tweaks/01-run.sh:20]: makes sense will switch

**Comment by mirceacaprioru** [comment]
changelog v1-> v2
- remove extra lines
- switch from cp to install

**Pull request merged**